docs/guides/latest/advanced-use-case/pause-backup.md

```diff
@@ -0,0 +1,365 @@
+Namespace:    demo
+Labels:       <none>
+Annotations:  kubectl.kubernetes.io/last-applied-configuration:
+                {"apiVersion":"stash.appscode.com/v1beta1","kind":"BackupConfiguration","metadata":{"annotations":{},"name":"pause-backup","namespace":"de...
+API Version:  stash.appscode.com/v1beta1
+Kind:         BackupConfiguration
+Metadata:
+  Creation Timestamp:  2019-08-01T11:42:22Z
+  Finalizers:
+    stash.appscode.com
+  Generation:        2
+  Resource Version:  14465
+  Self Link:         /apis/stash.appscode.com/v1beta1/namespaces/demo/backupconfigurations/pause-backup
+  UID:               5e1abe4b-c3d2-438a-ab8d-67ed7e663546
+Spec:
+  Paused:  true
+  Repository:
+    Name:  gcs-repo
+  Retention Policy:
+    Keep Last:  5
+    Name:       keep-last-5
+    Prune:      true
+  Schedule:     */1 * * * *
+  Target:
+    Directories:
+      /source/data
+    Ref:
+      API Version:  apps/v1
+      Kind:         Deployment
+      Name:         stash-demo
+    Volume Mounts:
+      Mount Path:  /source/data
+      Name:        source-data
+Events:
+  Type    Reason          Age   From                       Message
+  ----    ------          ----  ----                       -------
+  Normal  Backup Skipped  25s   Backup Triggering CronJob  Skipping creating BackupSession. Reason: Backup Configuration demo/pause-backup is paused.
+```
+
+Look at the `Events` of the `BackupConfiguration` crd. The event shows that the creation of `BackupSession` has paused. This means the hole backup process has stopped.
+
+**Skipped Instant Backup:**
+
+If we try to create a `BackupSession` for instant backup  the `BackupSession` will be `Skipped`. Now, we are going to create a `BackupSession` crd for instant backup. Below is the YAML of the `BackupSession` that we are going to create,
+
+```yaml
+apiVersion: stash.appscode.com/v1beta1
+kind: BackupSession
+metadata:
+  labels:
+    stash.appscode.com/backup-configuration: pause-backup
+  name: instant-backupsession
+  namespace: demo
+spec:
+  backupConfiguration:
+    name: pause-backup
+```
+
+let's create the `BackupSession` we have shown above.
+
+```console
+kubectl apply -f ./docs/examples/guides/latest/advanced-use-case/pause-backup/backupsession.yaml
+backupsession.stash.appscode.com/instant-backupsession created
+```
+
+Wait for next schedule for backup. Run the following command to watch `BackupSession` crd,
+
+```console
+Every 3.0s: kubectl get backupsession -n demo                      suaas-appscode: Thu Aug  1 19:10:39 2019
+
+NAMESPACE   NAME                       BACKUPCONFIGURATION   PHASE       AGE
+demo        instant-backupsession      pause-backup          Skipped     1m2s
+demo        pause-backup-1564659789    pause-backup          Succeeded   2m
+```
+
+So, we can see from the output of the above command that the restore process has Skipped. If you describe the `BackupSession` object you will see that there is a warning for skipping BackupSession.
```

Comment:
Notice the `PHASE` column. It is showing that the `BackupSession` has been skipped.

If you describe the `BackupSession` object you are going to see that there is an event explaining why it has been skipped.